packages/react/src/components/__tests__/__snapshots__/storyshots.test.js.snap

```diff
@@ -1,5 +1,274 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Storyshots Footer Default 1`] = `
```

Comment:
@kennylam @jeffchew What is this file? I didn't create this file. Should we be tracking it?

Reply:
@photodow Yes, this is a snapshot file that is autocreated from snapshot testing. Because it's a new story, it generates the snapshot and adds to this file. 

Unit tests would essentially fail if snapshots don't match up between the executed test and the saved snapshot file. You can update the snapshot file by running `test:updateSnapshot` if you know that the snapshot file needs to be updated.

cc: @kennylam @vlad-saling

Reply:
@photodow more info on snapshot testing in Jest: https://jestjs.io/docs/en/snapshot-testing